etelemetry/tests/test_client.py

```diff
@@ -4,6 +4,20 @@
 from ..client import _etrequest, get_project, check_available_version
 
 
+def check_cxn():
+    import requests
+
+    try:
+        requests.get("https://google.com")
```

Comment:
may be just

```suggestion
        requests.get("http://example.com")
```

- to use something "more standard" than a com company
- to not rely on SSL/https being all up to date etc
- be faster apparently:

```
❯ timeit curl --silent example.com
10 runs real:0.06+-1.46e-17 user:0+-0 sys:0.00+-5.16e-03
❯ timeit curl --silent w3c.org
10 runs real:0.33+-0.08 user:0+-0 sys:0.00+-0.00
❯ timeit curl --silent google.com
10 runs real:0.12+-0.02 user:0.00+-0.00 sys:0+-0
```